### PR TITLE
add duckling url env var

### LINKED ISSUE #36: no duckling url configured
in order to connect to duckling via ENABLE_DUCKLING=True, you need to give the right url in the configuration file. this should be done automatically in the setup.

### PR DESCRIPTION
**Pull request by erohmensing**
let's try this again 😄
- close #36

### CODE REVIEW

**Comment by erohmensing** [comment]
@wochinge probably something I just missed in the implementation, but can you help me track down the linting error?

File: charts/rasa-x/templates/rasa-deployments.yaml [hunk 1]
```diff
@@ -108,6 +108,8 @@ spec:
           value: "{{ .rasaEnvironment }}"
         - name: "RASA_MODEL_SERVER"
           value: "http://{{ include "rasa-x.host" $ }}:{{ $.Values.rasax.port }}/api/projects/default/models/tags/{{ .modelTag }}"
+        - name: "RASA_DUCKLING_HTTP_URL"
+          value: "http://{{ include "duckling.host" . }}:{{ $.Values.duckling.port }}"
```
Comment (wochinge) [inline charts/rasa-x/templates/rasa-deployments.yaml:112]:
```suggestion
          value: "http://{{ include "duckling.host" $ }}:{{ $.Values.duckling.port }}"
```

Comment (erohmensing) [reply charts/rasa-x/templates/rasa-deployments.yaml:112]: 🤦‍♀ thank you

Comment (wochinge) [reply charts/rasa-x/templates/rasa-deployments.yaml]: no facepalm please. That's a speciality of our chart and the for loop we are using in this file.

**Pull request merged**